pkg/scheduler/internal/cache/node_tree.go

```diff
@@ -127,6 +127,7 @@ func (nt *NodeTree) removeZone(zone string) {
 	for i, z := range nt.zones {
 		if z == zone {
 			nt.zones = append(nt.zones[:i], nt.zones[i+1:]...)
+			break
```

Comment:
You can return here instead of breaking.

Reply:
Thanks for the review.

I cannot find the branch for this PR.
Considering that the return and break are equivalent, can the current form be kept ?

If you still prefer return, I will create new PR

Reply:
> I cannot find the branch for this PR.

I don't buy it. Please don't create duplicate PRs that will take reviewers extra efforts.

Reply:
I might know what you mean. You shouldn't use `tedyu:master` to raise PR; instead, use `tedyu:<foo branch>` to compose changes and that should be where a PR is raised from. And your local `tedyu:master` branch should be always clean, and synced with upstream when needed.

Reply:
@Huang-Wei 
I am relatively new to k8s.
When I created this PR, I didn't anticipate further changes since this is a small PR.

I will follow best practice in the future so that such incident doesn't happen again.